internal/server/server.go

```diff
@@ -172,3 +174,31 @@ func (s ServerForClient) GetGameState() gamestate.ClientGameState {
 func (s ServerForClient) GetGameConfig() config.ClientConfig {
 	return s.server.gameConfig.GetClientConfig()
 }
+
+func randomAssign(input []shared.ClientID) (shared.ClientID, shared.ClientID, shared.ClientID) {
+	lst = make([]shared.ClientID, len(input))
```

Comment:
don't just commit changes btw--need to remove `arrayCopy`.

Reply:
I will